pyOCD/board/mbed_board.py

```diff
@@ -38,6 +38,8 @@
                 "1070": "nrf51822",
                 "1080": "stm32f103rc",
                 "1090": "stm32f051",
+                "1600": "lpc4330",
```

Comment:
They use two different IDs.  One for the Bambino 210 and another for the Bambino 210E.  This should support them both.
https://github.com/micromint/CMSIS-DAP-lpc43xx/blob/master/interface/Common/src/board.c#L77
https://github.com/micromint/CMSIS-DAP-lpc43xx/blob/master/interface/Common/src/board.c#L82